swingset/src/main/java/com/nqadmin/swingset/SSBaseComboBox.java

```diff
@@ -444,6 +446,51 @@ public void setSelectedOption(final O _option) {
 			setSelectedItem(item);
 		}
 	}
+	
+	/////////////////////////////////////////////////////////////////////////
+	//
+	// Deal with non-standard GlazedList UP/DOWN arrow handling
+	//
+	
+    /**
+     * When dealing with GlazedLists (1.11) restore expected combo behavior.
+     *
+     * Per GL JavaDoc:
+     * https://javadoc.io/doc/com.glazedlists/glazedlists/latest/ca/odell/glazedlists/swing/AutoCompleteSupport.html
+     *<p>
+     *  4. typing the up arrow key when the popup is visible and the selected element is the first element causes the autocompletion to be cleared and the popup's selection to be removed.
+     *  6. typing the down arrow key when the popup is visible and the selected element is the last element causes the autocompletion to be cleared and the popup's selection to be removed
+     *<p>
```

Comment:
Are there any issues if at begin/end and the popup is not visible?

Reply:
It seems to be working as expected. If the popup is not visible I don't think it would get the keystrokes.